Add instances to Cli and enable remote vs local run configurations (#26)
astriaorg/astria-cli-go: README.md

### Available Commands

| Command                    | Description                                                                         |

> | -------------------------- | ----------------------------------------------------------------------------------- |

Reviewer: What's this change for? Or was it from vscode autoformatting?

Author: Yeah this was just auto formatting. Not sure why it changed...

| `dev clean all`            | Deletes the local data, downloaded binaries, and config files for the Astria stack. |
| `sequencer create-account` | Generate an account for the Sequencer.                                              |
| `sequencer get-balance`    | Get the balance of an account on the Sequencer.                                     |

### Instances

> The `dev init`, `dev run`, and `dev clean` commands above all have an optional

Reviewer: ```suggestion
The `dev init`, `dev run`, and `dev clean` commands all have an optional
```

Reviewer: I try to refrain from referencing locations like this without a link because things will get moved around but they won't think to update other sections. I'd just remove the location reference.

> `--instance [string]` flag. Using the instance flag will "name" the config data
> created in the `~/.astria` directory, allowing you to have multiple, local
> configurations. If you don't use the flag, a `default` instance is created.

Reviewer: ```suggestion
`--instance` flag. The value of this flag will be used as the directory name where the rollup data will be stored. Now you can run many rollups while keeping their configs and state data separate. If no value is provided, the `default` is used, i.e. `~/.astria/default`.
```

For example, if you run:

```bash
astria-go dev init

> astria-go dev init -i hello

Reviewer: ```suggestion
astria-go dev init --instance hello
```
let's use long form in our examples to be explicit

astria-go dev init -i world
```

You will see the following in the `~/.astria` directory:

```bash
.astria/
    default/
    hello/
    world/
```

> Each of these directories will contain independent configs and binary files for

Reviewer: ```suggestion
Each of these directories will contain configs and binaries for
```

running the Astria stack. You can then update the `.env` files in the

> `.astria/<instance name>/config-local/` or `.astria/<instance

Reviewer: ```suggestion
`~/.astria/<instance name>/config-local/` or `~/.astria/<instance
```
